index.bs

```diff
@@ -4646,6 +4646,13 @@ Registry Definitions</h4>
 			The [=custodian=] may be the [=Working Group=], the [=Team=], or a delegated entity.
 			The [=custodian=] for all [=registry tables=] in a single [=registry=]
 			<em class=rfc2119>should</em> generally be the same entity.
+
+			If a [=custodian=] of a [=registry=] becomes defunct
+			(e.g., the relevant group is disbanded),
+			and the [=Working Group=] that owns the [=registry definition=] is itself also defunct,
```

Comment:
Incorporating all previous suggestions from @dwsinger, @nigelmegitt, and myself —
```suggestion
			If the [=custodian=] of a [=registry=] or [=registry table=]
			ceases to exist or to operate as a custodian
			(e.g., the relevant group is disbanded, or 
			the custodian is unresponsive to repeated attempts to make contact),
			and the [=Working Group=] that owns the [=registry definition=] 
			is itself <a href="#GeneralTermination">closed</a> or 
			unresponsive to repeated attempts to make contact,
```

I still prefer `or portion thereof` over `or [=registry table=]`, but I've not included it in this aggregate suggestion because nobody else has voiced a preference.

Reply:
I prefer it without "or portion thereof" - I don't think that phrase adds anything useful, unless there are some registries that have specific subsections whose custodians are different to the rest of the document, other than the registry table, and if that is the case, it seems strange and I'd like to know more about the use case.

Reply:
As I understand it, any `[=registry=]` may be defined to have multiple `[=registry tables=]`, and any subset of `[=registry tables=]` (1–_n_) might have a different `[=custodian=]` than any other subset, up to the extreme of each and every `[=registry table=]` within a `[=registry=]` having a different `[=custodian=]`, which might also differ from the `[=registry=]` itself.

I took the `SHOULD` in (lines 4647-4648 above) —
```
The [=custodian=] for all [=registry tables=] in a single [=registry=]
<em class=rfc2119>should</em> generally be the same entity.
```
— as an indication that there was already one or more `[=registries]` known or likely to be defined with different `[=custodians=]` for one or more `[=registry tables=]` therein.

I prefer `or portion thereof` because its the fewest words to allow for any subset (1–_n_) of the `[=registry tables=]` within a `[=registry=]` to be assigned to a new `[=custodian=]` without requiring that _all_ `[=registry tables=]` within that `[=registry=]`, or even the `[=registry=]` itself, be reassigned at the same time. I like this ability to "fix in part" because I can foresee time constraints being in play for the initial `[=registry table(s)=]` where the issue was discovered, while there is more time flexibility on the remaining `[=registry table(s)=]` and/or the `[=registry=]` itself.

Reply:
As discussed in today's call
```suggestion
			If the [=custodian=] of a [=registry table=]
			ceases to exist or to operate as a custodian
			(e.g., the relevant group is disbanded, or 
			the custodian is unresponsive to repeated attempts to make contact),
			and the [=Working Group=] that owns the [=registry definition=] 
			is itself <a href="#GeneralTermination">closed</a> or 
			unresponsive,
```